random/fn-call.json

```diff
@@ -0,0 +1,7 @@
+{
+  "code_file": "tmp/x.wasm",
+  "function": "rollDie",
+  "input": {
+    "sides": 20
```

Comment:
`die_sides?

Reply:
updated to `die_sides` here, in `main.go`, and README